ipaserver/install/httpinstance.py

```diff
@@ -272,21 +271,18 @@ def enable_mod_nss_ocsp(self):
 
         path = '/files{}/VirtualHost'.format(paths.HTTPD_NSS_CONF)
 
-        ocsp_comment = aug.get(
-                        '{}/#comment[.=~regexp("NSSOCSP .*")]'.format(path))
         ocsp_dir = aug.get('{}/directive[.="NSSOCSP"]'.format(path))
 
-        if ocsp_dir is None and ocsp_comment is not None:
-            # Directive is missing, comment is present
-            aug.set('{}/#comment[.=~regexp("NSSOCSP .*")]'.format(path),
-                    'NSSOCSP')
-            aug.rename('{}/#comment[.="NSSOCSP"]'.format(path), 'directive')
-        elif ocsp_dir is None:
-            # Directive is missing and comment is missing
-            aug.set('{}/directive[last()+1]'.format(path), "NSSOCSP")
+        # there is NSSOCSP directive in nss.conf file, comment it
+        # otherwise just do nothing
+        if ocsp_dir is not None:
+            ocsp_state = aug.get('{}/directive[.="NSSOCSP"]/arg'.format(path))
+            aug.remove('{}/directive[.="NSSOCSP"]/arg'.format(path))
+            aug.rename('{}/directive[.="NSSOCSP"]'.format(path), '#comment')
+            aug.set('{}/#comment[.="NSSOCSP"]'.format(path),
```

Comment:
Extracting the string literals into a separate variables named e.g. `ocsp_path`, `ocsp_arg` could really improve the readability of the method.